bt-target/client/main.lua

```diff
@@ -50,7 +50,12 @@ function playerTargetEnable()
 
                                         success = true
 
-                                        SendNUIMessage({response = "validTarget", data = Models[_]["options"]})
+                                        local options = Models[_]["options"];
```

Comment:
These semi-colons aren't valid lua syntax?

Reply:
Semicolons are valid Lua syntax. https://www.lua.org/pil/1.1.html They're optional and in this instance serve no purpose, but they are valid syntax.

Reply:
From my understanding from the documentation they're used to condense longer things into one line instead of line endings in JS or C, C# etc so this use case here is un-necessary and don't match the rest of the resource.

Reply:
That is one of their uses, yes.  They are rarely used in Lua, but are as optional as they are in JS.. so it's usually best to stay with the coding standards of the project.  Just wanted to point out the semicolon is a valid Lua syntax, however.